workflow/scripts/shared/eval_segmentation_paramsearch.py

```diff
@@ -0,0 +1,111 @@
+import pandas as pd
+from joblib import Parallel, delayed
+from pathlib import Path
+from lib.shared.eval_segmentation import evaluate_segmentation_paramsearch
+
+
+def get_file(filepath):
+    """Read TSV and add parameters from filename."""
+    try:
+        # Extract parameters from filename
+        path = Path(filepath)
+        prefix, params_part = path.stem.split("__")
+
+        # Extract well and tile from prefix
+        prefix_parts = prefix.split("_")
+        well = next(p.replace("W", "") for p in prefix_parts if p.startswith("W"))
+        tile = next(p.replace("T", "") for p in prefix_parts if p.startswith("T"))
+
+        # Extract other parameters from the second part
+        params = params_part.split("_")
+        nuclei_diameter = float(
+            next(p.replace("nd", "") for p in params if p.startswith("nd"))
+        )
+        cell_diameter = float(
+            next(p.replace("cd", "") for p in params if p.startswith("cd"))
+        )
+        flow_threshold = float(
+            next(p.replace("ft", "") for p in params if p.startswith("ft"))
+        )
+        cellprob_threshold = float(
+            next(p.replace("cp", "") for p in params if p.startswith("cp"))
+        )
```

Comment:
I've mentioned this before but if we want to store metadata in these/other files we should come up with a standardized way for us and other brieflow devs to do this. Just something to put in the backlog, not worth changing for this PR.

Reply:
You should keep track of all of these comments somewhere please! Idk if we need to open an issue for all of them, but let's keep these in mind.